package.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "@commitlint/cli": "^13.1.0",
     "@commitlint/config-conventional": "^13.1.0",
+    "chakra-ui-steps": "^1.5.0",
```

Comment:
Do we need this here??

Reply:
This must have been added when I ran `yarn add chakra-ui-steps` in an effort to fix dependency issues. Removed!